includes/admin/class-wc-rest-payments-refunds-controller.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * Class WC_REST_Payments_Timeline_Controller
+ *
+ * @package WooCommerce\Payments\Admin
+ */
+
+use WCPay\Core\Server\Request\Refund_Charge;
+use WCPay\Exceptions\API_Exception;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * REST controller for the timeline, which includes all events related to an intention.
+ */
```

Comment:
Can we add something like `@internal Not intended for usage in integrations or outside of WooCommerce Payments.` to flag the controller's intended usage?